### ISSUE TITLE
[Help] TiDE model output not within scale limits.

### ISSUE THREAD

**Issue by asw-v4** [unit8co/darts#2229]
**Describe the issue**
TiDE model is trained on targets, and past/future covariates, all scaled to 0:1. Model Validation Loss (MSE) decreases during training by a factor of 100-1000, suggesting it is learning (at least somewhat). I am using my own data (25 features measured over time, at 12 different locations)

**To Reproduce**
`targ` contains the stacked target feature TimeSeries, containing 12 components, , and `pCov` contains the stacked covariate features TimeSeries, 24 features each, 12 componants. Created as so - at freq 1H:
```python
            #... for loop per location (12)
            # create timeseries and append to list
            targetList.append(TimeSeries.from_dataframe(target))
            pCovList.append(TimeSeries.from_dataframe(pastCovariates))

        # concat all timeseries
        targets = concatenate(targetList, axis=1)
        pastCovariates = concatenate(pCovList, axis=1)
```
Viewing the value range (before scaling):
```bash
Min: component
Non-Tidal Residual      -1.511
Non-Tidal Residual_1    -1.301
....
Non-Tidal Residual_11    0.000

Max: component
Non-Tidal Residual       1.890
Non-Tidal Residual_1     1.663
....
Non-Tidal Residual_11    0.000
```

```python
        # split data and scale according to train set
        trainTarg, tmp = targ.split_after(model_env["DataSplits"]["train"])
        valTarg, testTarg = tmp.split_after(0.5)

        trainPCov, tmp = pCov.split_after(model_env["DataSplits"]["train"])
        valPCov, testPCov = tmp.split_after(0.5)

        scaler = Scaler()
        trainTarg = scaler.fit_transform(trainTarg) #Values between 1 and 0
        valTarg = scaler.transform(valTarg)
        testTarg = scaler.transform(testTarg)

        CovScaler = Scaler()
        trainPCov = CovScaler.fit_transform(trainPCov)
        valPCov = CovScaler.transform(valPCov)
        testPCov = CovScaler.transform(testPCov)
```

Now my code is set up to train multiple models at a time, but here is an example of the setup and fit()

```python
optimizer_kwargs = {
                "lr": model_env["learning_rate"], #1e-5
            }

            # early stopping (needs to be reset for each model later on)
            # this setting stops training once the the validation loss has not decreased by more than 1e-5 for 10 epochs
            early_stopping_args = {
                "monitor": "val_loss",
                "patience": 10,
                "min_delta": 1e-6,
                "mode": "min",
            }

            LLog = LossLogger()
            # PyTorch Lightning Trainer arguments
            pl_trainer_kwargs = {
                "gradient_clip_val": 1,
                "max_epochs": model_env["epochs"],
                "accelerator": "auto",
                "callbacks": [EarlyStopping(**early_stopping_args), LLog],
            }

            # learning rate scheduler
            lr_scheduler_cls = torch.optim.lr_scheduler.ExponentialLR
            lr_scheduler_kwargs = {
                "gamma": 0.999,
            }

            common_args = {
                "input_chunk_length": histLen,
                "output_chunk_length": predLen, #12 or 24 (horizon)
                "num_encoder_layers": 2,
                "num_decoder_layers": 2,
                "hidden_size": 256,
                "temporal_width_past": 32,
                "temporal_width_future": temporal_width_future, # 0 or predLen
                "decoder_output_dim": 1,  
                "temporal_decoder_hidden": 128, 
                "dropout": 0.1,
                "n_epochs": model_env["epochs"], 
                "loss_fn": model_env["loss_function"], # MSELoss()
                "optimizer_kwargs": optimizer_kwargs,
                "pl_trainer_kwargs": pl_trainer_kwargs,
                "lr_scheduler_cls": lr_scheduler_cls,
                "lr_scheduler_kwargs": lr_scheduler_kwargs,
                "batch_size": model_env["batch_size"], #1024
                "force_reset": True,
                "model_name": model_name,
                "work_dir": model_env["save_path"],
                "save_checkpoints": True,
            }
            model_env["model"] = TiDEModel(
                **common_args,
                use_reversible_instance_norm=RIN, # True or False
            )
            if "NoFuture" in model_name:
                # train model
                model_env["model"].fit(
                    series=trainTarg,
                    past_covariates=trainPCov,
                    val_series=valTarg,
                    val_past_covariates=valPCov,
                )
```

**Expected behavior**
on call of the predict(testTarg) method, receive an output of values between 0 and 1 (even if it is garbage accuracy)

**Actual Behavior** 
```bash
TiDE-NoFuture-24Hr

min: component
Non-Tidal Residual        -110.896235
Non-Tidal Residual_1     -6453.726626
....
Non-Tidal Residual_11     -924.997762

max: component
Non-Tidal Residual         607.469088
Non-Tidal Residual_1      4122.152038
....
Non-Tidal Residual_11      708.339625
```

**System (please complete the following information):**
 - Python version: 3.11.5
 - darts version 0.27.2

**Additional context**
At one point I was getting appropriate magnitude results on the output, but i cant for the life of me work out what has changed. This occurs regardless of how many epochs I train for. (I know the batch size is large, I have a lot of compute available - I get the same results when using a batch size of 32/64
